docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,467 @@
+  must provide the fee required for its processing).
+
+| Relay request - relay entry submission
+| The transaction submitting relay entry should have a predictable cost and the
+  gas cost of this transaction should be as low as possible, below 200k gas.
+
+| Relay request - relay entry submission
+| It should be possible to send low gas callbacks (up to the fixed gas cost
+  value) in the transaction submitting the relay entry.
+
+| Relay request - relay entry submission
+| Failure in the callback function should not revert the relay entry transaction.
+
+| Relay request - relay entry submission
+| The operation removing members from the sortition pool should be as cheap as
+  possible so that the member submitting relay entry does not have to pay any
+  additional costs for removing inactive members from the pool.
+
+| Relay request - relay entry timeouts
+| The soft relay entry timeout equals the group size multiplied by the number of
+  blocks for a member to become eligible to submit relay entry. The timer starts
+  at the moment when the first member becomes eligible (the moment when the
+  relay entry was requested). +
+  The hard relay entry timeout is a set value (measured in time units).
+
+| Relay request - relay entry timeouts
+| If no entry was successfully submitted until soft relay timeout is reached,
+  all operators in the group should start bleeding and losing their T stake. The
+  amount of lost stake should incrementally increase from 0 (at the moment of
+  soft relay entry timeout) to a fixed value (at the moment of hard relay entry
+  timeout).
+
+| Relay request - relay entry timeouts
+| If no entry was successfully submitted until soft relay timeout is reached,
+  all operators in the group should start bleeding and losing their T stake. The
+  amount of lost stake should incrementally increase from 0 (at the moment of
+  soft relay entry timeout) to a fixed value (at the moment of hard relay entry
+  timeout).
+
+| Relay request - relay entry timeouts
+| Anyone can send a transaction reporting hard relay entry timeout.
+
+| Relay request - relay entry timeouts
+| Upon noticing the hard relay entry timeout notification, the random beacon
+  contract should determine its validity.
+
+| Relay request - relay entry timeouts
+| When determining validity of the hard relay entry timeout notification, the
+  following aspects should be taken into account: +
+  - does it reference the existing relay entry request? (if not, then
+  notification invalid) +
+  - has the hard relay entry timeout passed? (if not, then notification invalid) +
+
+| Relay request - relay entry timeouts
+| If the hard relay entry timeout notification was deemed justified, then: +
+  - the signing group should get terminated +
+  - the process of choosing a signing group and tasking it with a relay entry
+  generation should be repeated.
+
+| Relay entry utilisation
+| Applications wanting to use a relay entry should submit another transaction
```

Comment:
Should we allow to easily read the recent relay entry?

Reply:
Hmm, I'm not sure what you mean by easily, but generally I believe that application should be able to retrieve the info about the recent entry. The UAC is taken straight from the [RFC-19](https://github.com/keep-network/keep-core/blame/main/docs/rfc/rfc-19-random-beacon-v2.adoc#L191).

Reply:
Sorry, I wasn't precise enough. I mean should the `RandomBeacon` expose a dedicated view function to check the `previousEntry`? Now the entry is emitted as an event which makes it possible to fetch but a bit problematic for off-chain clients. For on-chain clients, the entry is passed within the callback.

Reply:
Claarified in 6aa27e527b74c6bc8705469207369e9bf0979c7a.